core-util/Array.h

```diff
@@ -20,9 +20,9 @@
 
 #include <stddef.h>
 #include <stdint.h>
-#include "mbed-util/CriticalSectionLock.h"
-#include "mbed-util/PoolAllocator.h"
-#include "mbed-util/mbed-util.h"
+#include "core-util/CriticalSectionLock.h"
+#include "core-util/PoolAllocator.h"
+#include "core-util/mbed-util.h"
```

Comment:
...and all the `MBED_UTIL_` stuff inside the `mbed-util.h`. Renames are a lot trickier than they look.